doc/application/application.rst

```diff
@@ -161,53 +510,102 @@ build scripts like this:
 
    target_sources(app PRIVATE src/main.c)
 
-In the above code an existing library target named 'app' is being
-configured to have the source file src/main.c. The PRIVATE keyword
-indicates that we are modifying the internals of how the library is
-being built. Whereas using the keyword 'PUBLIC' would modify how other
-libraries that link with app are built. In this case, using PUBLIC
-here would cause libraries that link with app to also include the
-source file src/main.c, behaviour that we surely do not want. The
-'PUBLIC' keyword could however be useful when modifying the include
+In the above code, an existing library target named ``app`` is
```

Comment:
would you edit above to be ``:file:`CMakeLists.txt` ``

Reply:
No problem, will fix in a follow on.